### PR TITLE
TEST: modify tests for v2SignExpires

### PR DESCRIPTION
**Pull request by nicolas2bert**
This PR modifies signature V2 tests for https://github.com/scality/Arsenal/pull/193

### CODE REVIEW

**Comment by nicolas2bert** [comment]
@ironman-machine ok 
SCALITY_ARSENAL_BRANCH=COMPAT/v2SignExpires

**Comment by nicolas2bert** [comment]
@ironman-machine ok 
SCALITY_ARSENAL_BRANCH=COMPAT/v2SignExpires

**Comment by nicolas2bert** [comment]
@ironman-machine ok 
SCALITY_ARSENAL_BRANCH=COMPAT/v2SignExpires

File: tests/functional/aws-node-sdk/test/legacy/authV2QueryTests.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ describe('aws-node-sdk v2auth query tests', function testSuite() {
     // AWS allows an expiry further in the future
     itSkipAWS('should return an error code if expires header is too far ' +
         'in the future', done => {
-        const params = { Bucket: bucket, Expires: 3605 };
+        const params = { Bucket: bucket, Expires: 100001 };
```
Comment (LaurenSpiegel) [inline tests/functional/aws-node-sdk/test/legacy/authV2QueryTests.js]: Please add a comment what this time represents and what our time limit is so we remember.

File: tests/functional/aws-node-sdk/test/legacy/authV2QueryTests.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ describe('aws-node-sdk v2auth query tests', function testSuite() {
         });
 
     it('should create a bucket', done => {
-        const params = { Bucket: bucket, Expires: 3601 };
+        const params = { Bucket: bucket, Expires: 99990 };
```
Comment (LaurenSpiegel) [inline tests/functional/aws-node-sdk/test/legacy/authV2QueryTests.js]: please define 99990 as something like:
const almostOutsideTime = 99990 and let's use almostOutsideTime wherever we were using 3601 before so we will know what the numbers mean when we look back at this.

**Comment by nicolas2bert** [comment]
@ironman-machine ok 
SCALITY_ARSENAL_BRANCH=COMPAT/v2SignExpires

File: tests/functional/aws-node-sdk/test/legacy/authV2QueryTests.js [hunk 3]
```diff
@@ -38,9 +39,10 @@ describe('aws-node-sdk v2auth query tests', function testSuite() {
     });
 
     // AWS allows an expiry further in the future
+    // 100001 seconds is heigher that the Expires time limit: 100000 seconds
```
Comment (bennettbuchanan) [inline tests/functional/aws-node-sdk/test/legacy/authV2QueryTests.js]: Typo: 'heigher' should be 'higher'.

**Pull request merged**